composer.json

```
@@ -2,7 +2,7 @@
     "name": "olcs/olcs-utils",
     "description": "OLCS Utils",
     "require": {
-        "php": "^7.4",
+        "php": "^7.4 || ^8.0 || ^8.2 || ^8.3",
```

Could this be simplified? `^8.0` matches any `8.X` version already so the subsequent versions are duplicates.

```suggestion
        "php": "^7.4 || ^8.0",
```